README.md

```diff
@@ -1,22 +1,59 @@
 # Elasticsearch
 
-This repository contains Ruby integrations for [Elasticsearch](https://www.elastic.co/products/elasticsearch):
+This repository contains Ruby integrations for [Elasticsearch](https://www.elastic.co/products/elasticsearch).
 
-* a client for connecting to an Elasticsearch cluster,
-* a Ruby API for the Elasticsearch's REST API,
-* a Ruby API for the X-Pack APIs,
-* various extensions and utilities.
+The [`elasticsearch`](https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch)
+library is a wrapper for two separate libraries:
+
+* [`elasticsearch-transport`](https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch-transport),
+  which provides a low-level Ruby client for connecting to an [Elasticsearch](https://www.elastic.co/products/elasticsearch) cluster
```

Comment:
I don't think this link is needed. We linked to the same page with the same anchor text earlier.
```suggestion
  which provides a low-level Ruby client for connecting to an Elasticsearch cluster
```